care/facility/api/viewsets/file_upload.py

```diff
@@ -54,6 +55,16 @@ def get_serializer_class(self):
     def get_queryset(self):
         if "file_type" not in self.request.GET:
             raise ValidationError({"file_type": "file_type missing in request params"})
+
+        if self.request.user.user_type in (
```

Comment:
Maybe switch this to a permission check, kinda misplaced in the query set. A new Permission Check would be great.